microprofile-rest-client/README.adoc

```diff
@@ -23,27 +24,38 @@ In this quickstart, we have two services -- a country server and a country clien
 server provides a simple REST interface providing information about some countries. The
 client consumes this API through the MicroProfile REST Client specification.
 
-== Solution
-
-We recommend that you follow the instructions in the next sections and create the
-application step by step. However, you can go right to the completed example which
-is available in this directory.
+// System Requirements
+include::../shared-doc/system-requirements.adoc[leveloffset=+1]
 
-. Start your {productName} server
+// Use of {jbossHomeName}
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
 
 // Start the {productName} Standalone Server
 include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
 
-. Compile the application code and deploy it to {productName}
-[source,options="nowrap"]
-----
-$ mvn clean package wildfly:deploy
-----
+== Solution
+
+We recommend that you follow the instructions that
+<<creating-new-project, create the application step by step>>. However, you can
+also go right to the completed example which is available in this directory.
+
+// Build and Deploy the Quickstart
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
 
-Run the above command first in the `country-server` directory and then in the
+// Undeploy the Quickstart
+include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+1]
+
+IMPORTANT: Run the above commands first in the `country-server` directory and then in the
```

Comment:
I think this should go one section above so it is included within deploying. User can read the undeploy section later.